frontend/src/model/Player.ts

```diff
@@ -0,0 +1,80 @@
+import Technologies from '../constants/technologies';
+import { defaultTechnology, defaultPlayersMeepleCount } from '../constants/gameDefaults';
+
+// temporary classes
+class Tile {}
+class Meeple {
+  player: Player;
+  constructor(player: Player) {
+    this.player = player;
+  }
+}
+// end of temporary classes
+
+class Player {
+  public name: string;
+  public technology: Technologies;
+  private score: number;
+  private placedTiles: Tile[];
+  private meepleList: Meeple[];
+
+  constructor(name: string, technology: Technologies = defaultTechnology) {
+    this.name = name;
+    this.technology = technology;
+
+    this.initializePlayerState();
+  }
+
+  initializePlayerState() {
+    this.score = 0;
+    this.placedTiles = [];
+    this.meepleList = [];
+
+    for (let i = 0; i < defaultPlayersMeepleCount; i++) {
+      this.meepleList.push(new Meeple(this));
+    }
```

Comment:
I would leave this part of this method and maybe call it - initializeMeeples. In the future it will be reworked to handle different types of Meeples

Reply:
Done